src/google/protobuf/compiler/csharp/csharp_group_field.cc

```diff
@@ -0,0 +1,188 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Both c++ and java do not have generators for group:
https://github.com/protocolbuffers/protobuf/tree/ba8692fbade4ba329cc4531e286ab5a8e0821d97/src/google/protobuf/compiler/java
https://github.com/protocolbuffers/protobuf/tree/ba8692fbade4ba329cc4531e286ab5a8e0821d97/src/google/protobuf/compiler/cpp

The group can share code gen with messages:
https://github.com/protocolbuffers/protobuf/blob/ba8692fbade4ba329cc4531e286ab5a8e0821d97/src/google/protobuf/compiler/java/java_helpers.cc#L373